source/features/visit-tag.tsx

```diff
@@ -10,6 +11,11 @@ import {wrapAll} from '../helpers/dom-utils.js';
 import {buildRepoURL} from '../github-helpers/index.js';
 
 async function addLink(branchSelector: HTMLButtonElement): Promise<void> {
+	// If the branch picker is open, do nothing #7491
+	if (elementExists('#selectPanel')) {
```

Comment:
Could work but it's not great. Do you see anyway to tell whether we are on a tag without relying on the picker?

https://github.com/refined-github/refined-github/tree/24.6.25

Reply:
The only thing of interest I can find (on current GitHub.com) is some SSR hydration data:

```js
// On `isRepoRoot`
data = document.querySelector('[partial-name="repos-overview"] [data-target="react-partial.embeddedData"]').text
payload = JSON.parse(data).props.initialPayload

// On `isRepoTree`
data = document.querySelector('react-app [data-target="react-app.embeddedData"]').text
payload = JSON.parse(data).payload

payload.refInfo.refType // `branch` or `tag`
```

I'm not sure how reliable this is though (whether the DOM selector or JSON selector).